# HG changeset patch
# User Andreas Weber <octave@josoansi.de>
# Date 1546467142 -3600
#      Wed Jan 02 23:12:22 2019 +0100
# Branch stable
# Node ID 8707e1a4a71bb25571cd96a36ff8430c7be1c66c
# Parent  3f367754063469daaa9a7c852a5af89ecd587609
signal-wrappers.c: Fix static analyzer detected V517 issue (bug #55347)

remove redundant check for SIGINT

diff --git a/liboctave/wrappers/signal-wrappers.c b/liboctave/wrappers/signal-wrappers.c
--- a/liboctave/wrappers/signal-wrappers.c
+++ b/liboctave/wrappers/signal-wrappers.c
@@ -166,23 +166,16 @@ octave_get_sig_number (const char *signa
     }
   else if (! strcmp (signame, "SIGINFO"))
     {
 #if defined (SIGINFO)
       *signum = SIGINFO;
       return true;
 #endif
     }
-  else if (! strcmp (signame, "SIGINT"))
-    {
-#if defined (SIGINT)
-      *signum = SIGINT;
-      return true;
-#endif
-    }
   else if (! strcmp (signame, "SIGIOT"))
     {
 #if defined (SIGIOT)
       *signum = SIGIOT;
       return true;
 #endif
     }
   else if (! strcmp (signame, "SIGLOST"))
